Marlin/Conditionals_LCD.h

```diff
@@ -101,7 +110,8 @@
   #if ENABLED(ULTIMAKERCONTROLLER)              \
    || ENABLED(REPRAP_DISCOUNT_SMART_CONTROLLER) \
    || ENABLED(G3D_PANEL)                        \
-   || ENABLED(RIGIDBOT_PANEL)
+   || ENABLED(RIGIDBOT_PANEL)                   \
+   || ENABLED(REPRAPWORLD_KEYPAD)
```

Comment:
It looks to me like ANET_LCD_KEYPAD needs both REPRAPWORLD_KEYPAD and NEWPANEL, but enabling REPRAPWORLD_KEYPAD will enable NEWPANEL for us, so we don't need to explicitly enable NEWPANEL in the `#if ENABLED(ANET_KEYPAD_LCD)` block.